Library/Formula/procmail.rb

```diff
@@ -0,0 +1,78 @@
+class Procmail < Formula
+  desc "procmail: sort incoming mail into separate folders/files"
+  homepage "http://www.procmail.org/"
+  url "ftp://ftp.procmail.net/pub/procmail/procmail-3.22.tar.gz"
+  mirror "ftp://ftp.ucsb.edu/pub/mirrors/procmail/procmail-3.22.tar.gz"
+  sha256 "087c75b34dd33d8b9df5afe9e42801c9395f4bf373a784d9bc97153b0062e117"
+
+  depends_on "autoconf" => :build
+
+  patch :p1, :DATA
```

Comment:
Please add a comment explaining why this patch is necessary as well as a link to the upstream submission. Thanks!